src/redux/orders/orders.operations.js

```diff
@@ -1,7 +1,249 @@
+import {getItems, setItems, client} from '../../utils/client'
 import { gql } from '@apollo/client';
-import { client, getItems } from '../../utils/client';
 
-const getAllOrders = async (skip, limit) => {
+export const getOrderById = (id) => {
+  const query = `
+    query getOrder($id:ID!){
+  getOrderById(id: $id) {
+    ...on Order {
+      _id
+      status
+      user {
+        firstName
+        lastName
+        patronymicName
+        email
+        phoneNumber
+      }
```

Comment:
Maybe it's better to assign this query to some variable, thus method will become more readable